libs/designsystem/menu/src/menu.component.ts

```diff
@@ -41,13 +44,13 @@ export class MenuComponent implements AfterViewInit {
 
   @Input() public triggers: Array<TriggerEvent> = ['click'];
 
-  @Input() public DOMPortalOutlet: HTMLElement | undefined;
+  @Input() public DOMPortalOutlet: HTMLElement = this.elementRef.nativeElement.ownerDocument.body;
```

Comment:
We should update the API table with this new default - and possibly also do a small rewrite of the Portal docs now that it is the default.

Reply:
Good catch! Fixed:

![image](https://user-images.githubusercontent.com/9556693/234577474-2e677218-6b25-4a6d-a56b-e3f9e332506c.png)